app/javascript/spec/select/__snapshots__/select.spec.js.snap

```diff
@@ -343,146 +324,147 @@ exports[`Select component should match the snapshot 1`] = `
+                              Object {
+                                "label": "Cats",
+                                "value": "2",
+                              },
+                            ],
+                            "pageSize": 5,
+                            "placeholder": "<Choose>",
+                            "screenReaderStatus": [Function],
+                            "styles": Object {},
+                            "tabIndex": "0",
+                            "tabSelectsValue": true,
+                            "value": Array [],
+                          }
                         }
-                      }
-                    >
-                      <ForwardRef(render)
-                        __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
-                        className="ddorg__pf3-component-mapper__select"
-                        css={
+                        setValue={[Function]}
+                        theme={
                           Object {
-                            "boxSizing": "border-box",
-                            "direction": null,
-                            "label": "container",
-                            "pointerEvents": null,
-                            "position": "relative",
+                            "borderRadius": 4,
+                            "colors": Object {
+                              "danger": "#DE350B",
+                              "dangerLight": "#FFBDAD",
+                              "neutral0": "hsl(0, 0%, 100%)",
+                              "neutral10": "hsl(0, 0%, 90%)",
+                              "neutral20": "hsl(0, 0%, 80%)",
+                              "neutral30": "hsl(0, 0%, 70%)",
+                              "neutral40": "hsl(0, 0%, 60%)",
+                              "neutral5": "hsl(0, 0%, 95%)",
+                              "neutral50": "hsl(0, 0%, 50%)",
+                              "neutral60": "hsl(0, 0%, 40%)",
+                              "neutral70": "hsl(0, 0%, 30%)",
+                              "neutral80": "hsl(0, 0%, 20%)",
+                              "neutral90": "hsl(0, 0%, 10%)",
+                              "primary": "#2684FF",
+                              "primary25": "#DEEBFF",
+                              "primary50": "#B2D4FF",
+                              "primary75": "#4C9AFF",
+                            },
+                            "spacing": Object {
+                              "baseUnit": 4,
+                              "controlHeight": 38,
+                              "menuGutter": 8,
+                            },
                           }
                         }
-                        onKeyDown={[Function]}
                       >
-                        <div
-                          className="ddorg__pf3-component-mapper__select css-2b097c-container"
+                        <ForwardRef(render)
+                          __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
```

Comment:
> __EMOTION_TYPE_PLEASE_DO_NOT_USE__

![tenor](https://user-images.githubusercontent.com/289743/102082169-62e90c00-3e09-11eb-83a9-b6e4674469ce.png)

Reply:
cc @Hyperkid123

Reply:
:rofl:

Reply:
I didn't mean this as a joke :laughing: this is coming from DDF

Reply:
Yeah and that is coming from PF3. We don't use any custom JSX. :upside_down_face:

Reply:
I *think* it just means [emotion](https://github.com/emotion-js/emotion) is using a private helper field, and calling it "do not use" to prevent *others* from using that value. But fun :)